Bottom: 04318cd780cf11ee0d2b4388ba075fcd9d9e7e93
Top:    e8f6dee564775554f90edc614f9fa03456ac3d08
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:08 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 017fbf4..33fe6c3 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1295,6 +1295,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
